lib/sandthorn_sequel_projection.rb

```diff
@@ -22,7 +21,7 @@ class << self
     require 'delegate'
     extend Forwardable
 
-    def_delegators :configuration, :batch_size, :event_stores
+    def_delegators :configuration, :batch_size, :event_store
```

Comment:
The events are in order in a event store and not between two. But I can be missing some specifics why it would be interesting to have more than one event store / projection.